Lib/test/_test_multiprocessing.py

```diff
@@ -337,6 +337,59 @@ def handler(*args):
         if os.name != 'nt':
             self.assertEqual(p.exitcode, -signal.SIGTERM)
 
+    def test_kill(self):
```

Comment:
It looks like this is duplicating most of `test_terminate`. Perhaps those two methods could use a shared helper?